<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAdding</class>
 <widget class="QDialog" name="DialogAdding">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>941</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="laptop_ID"/>
    </item>
    <item>
     <widget class="QLineEdit" name="Company"/>
    </item>
    <item>
     <widget class="QLineEdit" name="Product"/>
    </item>
    <item>
     <widget class="QLineEdit" name="TypeName"/>
    </item>
    <item>
     <widget class="QLineEdit" name="Inches"/>
    </item>
    <item>
     <widget class="QLineEdit" name="ScreenResolution"/>
    </item>
    <item>
     <widget class="QLineEdit" name="Cpu"/>
    </item>
    <item>
     <widget class="QLineEdit" name="Ram"/>
    </item>
    <item>
     <widget class="QLineEdit" name="Memory"/>
    </item>
    <item>
     <widget class="QLineEdit" name="Gpu"/>
    </item>
    <item>
     <widget class="QLineEdit" name="OpSys"/>
    </item>
    <item>
     <widget class="QLineEdit" name="Weight"/>
    </item>
    <item>
     <widget class="QLineEdit" name="Price_euros"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>240</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
